canvas_modules/common-canvas/webpack.config.js

```diff
@@ -55,7 +55,8 @@ module.exports = {
 		"lib/context-menu": "./src/context-menu/context-menu-wrapper.jsx",
 		"lib/command-stack": "./src/command-stack/command-stack.js",
 		"lib/canvas": "./src/common-canvas/index.js",
-		"common-canvas": "./src/index.js" // needs to be last to create correct combined css output
+		"common-canvas": "./src/index.js",
+		"styles/common-canvas": "./src/index.scss"
```

Comment:
This ends up being a breaking change.  The css is now saved in a styles directory.